demos/microcloud-deploy/variables.tf

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Required newline at the beginning of the file?

Reply:
removed